ScreenManager.lua

```diff
@@ -97,6 +97,11 @@ local function validateScreen( screen )
     end
 end
 
+---
+-- No operation function
+--
+local function null() end
```

Comment:
This may also be moved inside the function (but I would declare it as a local variable anyway, since it's used in a loop)